homeassistant/components/shelly/entity.py

```diff
@@ -20,6 +20,33 @@ def temperature_unit(block_info: dict) -> str:
     return TEMP_CELSIUS
 
 
+def shelly_naming(self, block, entity_type: str):
```

Comment:
It's standard to use a verb in imperative as first word in function names. I'd rename this to `name_shelly`.